content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/deploy-corda-cluster.md

```diff
@@ -52,20 +52,20 @@ The Corda Docker images must be in a Docker registry that is accessible from the
 
 ## Download Helm Charts
 
-1. Download the Helm charts `tar` file from the [R3 Customer Hub](https://r3.force.com/).
+* Download the Helm charts `corda-5.0.0-Eagle.tgztar` file from the [R3 Customer Hub](https://r3.force.com/).
```

Comment:
should be wither `corda-5.0.0-Eagle.tgz` or `corda-5.0.0-Eagle.tar.gz` - @ronanbrowne please ack